test/createEpicMiddleware-spec.js

```diff
@@ -190,57 +175,6 @@ describe('createEpicMiddleware', () => {
     }]);
   });
 
-  it('should warn about accessing state$.value before @@redux/INIT', () => {
```

Comment:
Is there a way to keep this? It works now and there is probably a proper use case for it but it's going to cause confusion. We should add it to the FAQ at least

Reply:
ohhh like keep the test but invert it? That you _can_ access `state$.value` on epic startup because INIT has already been passed through reducers?

Reply:
I went ahead and added that. lmk if you actually wanted something else

Reply:
oh, I don't want to warn because this is now perfectly valid. It's true that often it might be a mistake, but I'm not sure we can both offer it as valid _and_ warn?

Reply:
Gonna merge as we can introduce the warning again if we figure out a good way. Definitely interested in continuing this discussion

Reply:
Similar situation as the infinite loop then. Will need to add it to the FAQ